.github/workflows/ci-cd.yml

```diff
@@ -0,0 +1,73 @@
+name: ci-cd
+
+on: [push, pull_request]
+
+jobs:
+  #  ci:
+  #    # Set up operating system
+  #    runs-on: ubuntu-latest
+  #
+  #    # Define job steps
+  #    steps:
+  #      - name: Set up Python
+  #        uses: actions/setup-python@v4
+  #        with:
+  #          python-version: "3.9"
+  #
+  #      - name: Check-out repository
+  #        uses: actions/checkout@v3
+  #
+  #      - name: Install poetry
+  #        uses: snok/install-poetry@v1
+  #
+  #      - name: Install package
+  #        run: poetry install
+  #
+  #      - name: Test with pytest
+  #        run: poetry run pytest tests/ --cov=food_price_tracker --cov-report=xml
+  #
+  #      - name: Use Codecov to track coverage
+  #        uses: codecov/codecov-action@v3
+  #        with:
+  #          files: ./coverage.xml   # coverage report
+  #
+  #      - name: Build documentation
+  #        run: poetry run make html --directory docs/
+
+  cd:
+    permissions:
+      id-token: write
+      contents: write
+    
+        #    # Only run this job if the "ci" job passes
+        #    needs: ci
+
+    # Only run this job if new work is pushed to "main"
+    if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+
+    # Set up operating system
+    runs-on: ubuntu-latest
+
+    # Define job steps
+    steps:
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.9"
```

Comment:
What python version are we going to use? Here is defaulted as 3.9. 
Could move up to 3.12 if desired, but 3.9 is still supported till 2025